src/neuroconv/tools/nwb_helpers/_configuration_models/_base_backend.py

```diff
@@ -13,9 +14,11 @@
 class BackendConfiguration(BaseModel):
     """A model for matching collections of DatasetConfigurations to a specific backend."""
 
-    backend: ClassVar[Literal["hdf5", "zarr"]] = Field(
-        description="The name of the backend used to configure the NWBFile."
-    )
+    backend: ClassVar[Literal["hdf5", "zarr"]]
```

Comment:
So, we had talked about whether or not this should be a property of the model instead of a class attribute

However, especially after the `from_*` class constructors, our treatment of these fields become solidified as class attributes, which feels right to me